targets/TARGET_STM/TARGET_STM32F4/STM32Cube_FW/system_stm32f4xx.c

```diff
@@ -147,7 +166,7 @@ const uint8_t APBPrescTable[8]  = {0, 0, 0, 0, 1, 2, 3, 4};
   * @param  None
   * @retval None
   */
-__weak void SystemInit(void)
+void SystemInit(void)
```

Comment:
__weak removed - breaks custom targets with custom initialization

Reply:
@JeanMarcR 
Can you add it back in #14548 ?

Reply:
How did this happen, @jeromecoutant ? Is the system init not weak in the drivers or is this done only for Mbed ?